Source/Events/Relativity/Consumer/QuantumTunnelConnection.cs

```diff
@@ -88,122 +86,104 @@ public class QuantumTunnelConnection : IDisposable
             _tenantOffsetRepository = tenantOffsetRepository;
             _tenants = tenants;
 
             AppDomain.CurrentDomain.ProcessExit += ProcessExit;
             AssemblyLoadContext.Default.Unloading += AssemblyLoadContextUnloading;
-            Task.Run(() => Run(), _runCancellationToken);
-            Console.CancelKeyPress += (s, e) => Close();
+            Task.Run (() => Run (), _runCancellationToken);
+            Console.CancelKeyPress += (s, e) => Close ();
         }
 
         /// <summary>
         /// Destructs the <see cref="QuantumTunnelConnection"/>
         /// </summary>
-        ~QuantumTunnelConnection()
-        {
-            Dispose();
+        ~QuantumTunnelConnection () {
+            Dispose ();
         }
 
         /// <inheritdoc/>
-        public void Dispose()
-        {
+        public void Dispose () {
             AppDomain.CurrentDomain.ProcessExit -= ProcessExit;
             AssemblyLoadContext.Default.Unloading -= AssemblyLoadContextUnloading;
 
-            Close();
+            Close ();
         }
 
-        void ProcessExit(object sender, EventArgs e)
-        {
-            Close();
+        void ProcessExit (object sender, EventArgs e) {
+            Close ();
         }
 
-        void AssemblyLoadContextUnloading(AssemblyLoadContext context)
-        {
-            Close();
+        void AssemblyLoadContextUnloading (AssemblyLoadContext context) {
+            Close ();
         }
 
-        void Close()
-        {
-            _runCancellationTokenSource.Cancel();
+        void Close () {
+            _runCancellationTokenSource.Cancel ();
 
             //if( _runThread != null ) _runThread.Abort();
 
-            _logger.Information("Collapsing quantum tunnel");
-            _channel.ShutdownAsync();
+            _logger.Information ("Collapsing quantum tunnel");
+            _channel.ShutdownAsync ();
         }
 
-        void Run()
-        {
-            _logger.Information($"Establishing connection towards event horizon for application ('{_destinationKey.Application}') and bounded context ('{_destinationKey.BoundedContext}') at '{_url}'");
+        void Run () {
+            _logger.Information ($"Establishing connection towards event horizon for application ('{_destinationKey.Application}') and bounded context ('{_destinationKey.BoundedContext}') at '{_url}'");
 
-            Task.Run(async() =>
-            {
+            Task.Run (async () => {
                 _runThread = Thread.CurrentThread;
-                for(;;)
-                {
-                    
-                    _runCancellationToken.ThrowIfCancellationRequested();
-
-                    try
-                    {
-                        await OpenAndHandleStream();
-                    }
-                    catch (Exception ex)
-                    {
-                        _logger.Error(ex, "Error occurred during establishing quantum tunnel");
+                for (;;) {
```

Comment:
In C# we don't have our curlies on the same line - but the next line

Reply:
Ahh, I did the "Format Document" command for some of the files, that command apparently did this for some reason...